--- fastcommunity_mh.cc.0	2010-09-01 20:20:12.660559910 +0900
+++ fastcommunity_mh.cc	2010-09-01 20:21:06.797511820 +0900
@@ -71,9 +71,10 @@
 //
 ////////////////////////////////////////////////////////////////////////
 
-#include <iostream.h>
+#include <iostream>
 #include <fstream>
 #include <string>
+#include <string.h>
 #include "stdlib.h"
 #include "time.h"
 #include "math.h"
--- maxheap.h.0	2010-09-01 20:20:28.164601525 +0900
+++ maxheap.h	2010-09-01 20:20:54.906496228 +0900
@@ -40,7 +40,8 @@
 #if !defined(MAXHEAP_INCLUDED)
 #define MAXHEAP_INCLUDED
 
-#include <iostream.h>
+#include <iostream>
+using namespace std;
 
 /*   Because using this heap requires us to be able to modify an arbitrary element's
 	data in constant O(1) time, I use to tricky tactic of having elements in an array-
--- vektor.h.0	2010-09-01 20:20:38.597760767 +0900
+++ vektor.h	2010-09-01 20:20:42.582548367 +0900
@@ -30,7 +30,7 @@
 #if !defined(vektor_INCLUDED)
 #define vektor_INCLUDED
 
-#include <iostream.h>
+#include <iostream>
 
 #if !defined(vektor_INCLUDED)
 #define vektor_INCLUDED
